src/radical/pilot/agent/lm/base.py

```diff
@@ -86,6 +87,7 @@ def __init__(self, name, cfg, session):
         self.env_removables = []
 
         self._configure()
+        # pylint: enable=protected-access
```

Comment:
It's a shame that python has no granularity beyond `public/private`...  The session logger is the right thing to use here, but the logger should not be part of the application facing API.  Something like a C++ `friend` is what we would need here.  So we add two linter lines for one code line :-P  Oh well...